config/target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+8
```

Comment:
Make sure you pass all the tests for score 8 and get green PR before the assignment.